connect-inject/container_init.go

```diff
@@ -29,15 +32,16 @@ type initContainerCommandData struct {
 	// EnableMetrics adds a listener to Envoy where Prometheus will scrape
 	// metrics from.
 	EnableMetrics bool
-	// todo: remove once endpoints controller supports metrics
-	// PrometheusScrapeListener configures the listener on Envoy where
-	// Prometheus will scrape metrics from.
-	PrometheusScrapeListener string
```

Comment:
this was also unused once we added support for metrics merging to the endpoints controller